portfolio/src/main/webapp/script.js

```diff
@@ -62,7 +71,16 @@ function visitLink(s){
 
 //Writes a string of html
 function writeHTMLString(b){
-    htmlString = '<div id = "board">';
+    let htmlString = '';
+    htmlString += '<div id = "players-container"></div>';
+    htmlString += '<form action = "/data" method="POST" id = "players-form">';
+    htmlString += '<p>Player One:</p>';
+    htmlString += '<input type = "text" name="p1" value = "PlayerOne"> <br />';
+    htmlString += '<p>Player Two:</p>';
+    htmlString += '<input type = "text" name = "p2" value = "PlayerTwo">';
+    htmlString += '<br /><br />';
+    htmlString += '<input type = "submit" class = "button"/></form>';
```

Comment:
I don't think I understand why this part is part of the method vs statically present in the html. or maybe present but with visiblity toggled as needed. I think, in general, you should try to avoid. Creating HTML in the JS if it isn't required. It might make it harder for other developers to find where the HTML is to make fixes. Also it is easier to make errors when it is wrapped in a string because linters will ignore them.